### DISCUSSION TITLE
compile PETSc on cluster

### DISCUSSION THREAD

**Discussion by wsfsz123** [idaholab/moose#16330] [Q&A Getting Started] [answered]
Hi, dear all,

I am always stacked when compiling PETSc using the code ./scripts/update_and_rebuild_petsc.sh on cluster, as shown in the attached picture. I don't know how to fix it, I am following the steps on
 https://mooseframework.inl.gov/getting_started/installation/hpc_install_moose.html

Thanks,
Shizhe

![image](https://user-images.githubusercontent.com/35913685/100582451-353d9680-3324-11eb-903f-ef72f2fe6802.png)

**Comment by milljm** [comment]
What you are seeing is what you should be seeing. There is no error here.

**Reply by dschwen** [reply]
Maybe "stacked" means "stuck" here, and the hypre download is not completing. This is odd, because the PETSc config script appears to have successfully downloaded SUPERLU from github already. Try giving it more time.

**Comment by milljm** [comment]
I received notification emails that you posted (twice) to this discussion, but I am not seeing those posts. From what I am reading (in my email I _did_ receive from GitHub), it would seem PETSc may not have successfully finished building. Can you post the configure.log for PETSc as well? That should reside at:

```pre
moose/petsc/configure.log
```

Thanks!

**Comment by wsfsz123** [comment, marked as answer]
Thank you all. I have solved this problem, it seems I had trouble git hypre, I downloaded it manually and modified the configure code, it works.
